.vscode/settings.json

```diff
@@ -77,4 +77,12 @@
   "[terraform]": {
     "editor.defaultFormatter": "hashicorp.terraform"
   }
+  "editor.tabSize": 2,
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "files.insertFinalNewline": true,
+  "files.trimFinalNewlines": true,
+  "css.validate": false,
+  "editor.wordWrap": "wordWrapColumn",
+  "editor.wordWrapColumn": 120
```

Comment:
If we set Prettier as the formatter, why do we need to specify `editor.` specific settings? It will conflict with whatever might be set in `.prettierrc`.